src/registrar/forms/portfolio.py

```diff
@@ -110,102 +109,234 @@ def clean(self):
-
-
-class PortfolioInvitedMemberForm(forms.ModelForm):
-    """
-    Form for updating a portfolio invited member.
-    """
-
-    roles = forms.MultipleChoiceField(
-        choices=UserPortfolioRoleChoices.choices,
-        widget=forms.SelectMultiple(attrs={"class": "usa-select"}),
-        required=False,
-        label="Roles",
-    )
-
-    additional_permissions = forms.MultipleChoiceField(
-        choices=UserPortfolioPermissionChoices.choices,
-        widget=forms.SelectMultiple(attrs={"class": "usa-select"}),
-        required=False,
-        label="Additional Permissions",
-    )
-
-    class Meta:
-        model = PortfolioInvitation
-        fields = [
-            "roles",
-            "additional_permissions",
-        ]
-
-
-class NewMemberForm(forms.ModelForm):
-    member_access_level = forms.ChoiceField(
-        label="Select permission",
-        choices=[("admin", "Admin Access"), ("basic", "Basic Access")],
-        widget=forms.RadioSelect(attrs={"class": "usa-radio__input  usa-radio__input--tile"}),
+class BasePortfolioMemberForm(forms.Form):
+    required_star = '<abbr class="usa-hint usa-hint--required" title="required">*</abbr>'
+    role = forms.ChoiceField(
+        choices=[
+            (UserPortfolioRoleChoices.ORGANIZATION_ADMIN.value, "Admin access"),
+            (UserPortfolioRoleChoices.ORGANIZATION_MEMBER.value, "Basic access"),
+        ],
+        widget=forms.RadioSelect,
         required=True,
         error_messages={
             "required": "Member access level is required",
         },
     )
-    admin_org_domain_request_permissions = forms.ChoiceField(
-        label="Select permission",
-        choices=[("view_only", "View all requests"), ("view_and_create", "View all requests plus create requests")],
+
+    domain_request_permission_admin = forms.ChoiceField(
+        label=mark_safe(f"Select permission {required_star}"),  # nosec
+        choices=[
+            (UserPortfolioPermissionChoices.VIEW_ALL_REQUESTS.value, "View all requests"),
+            (UserPortfolioPermissionChoices.EDIT_REQUESTS.value, "View all requests plus create requests"),
+        ],
         widget=forms.RadioSelect,
-        required=True,
+        required=False,
         error_messages={
             "required": "Admin domain request permission is required",
         },
     )
-    admin_org_members_permissions = forms.ChoiceField(
-        label="Select permission",
-        choices=[("view_only", "View all members"), ("view_and_create", "View all members plus manage members")],
+
+    member_permission_admin = forms.ChoiceField(
+        label=mark_safe(f"Select permission {required_star}"),  # nosec
+        choices=[
+            (UserPortfolioPermissionChoices.VIEW_MEMBERS.value, "View all members"),
+            (UserPortfolioPermissionChoices.EDIT_MEMBERS.value, "View all members plus manage members"),
+        ],
         widget=forms.RadioSelect,
-        required=True,
+        required=False,
```

Comment:
(Change Request): I know we went over the different approaches of subtracting vs adding error messages during validation.  I think both have merits depending on the situation.  For this one, since we are handling the population of error messages, it would be helpful to include a comment here to explain why we are setting required=False, but yet still including a "required" error message.  Just let other coders know where to look next if they ever need to set this to be truly not required.